setup.py

```diff
@@ -2,7 +2,7 @@
 from distutils.core import setup
 
 
-VERSION = '0.0.1'
+VERSION = '1.0.0'
```

Comment:
How do you feel about doing a minor release instead of a major one (i.e. 0.1.0) -- that way you don't imply stability yet (usually the first major release means it is stable), but it is major enough for distribution.

Reply:
That's probably a really good idea. I'll update the number.